2025.6 support

ADeadPixel opened the pull request:
reverted back to my original patch

ADeadPixel committed:
2025.6 support

adamoutler commented:
This PR modifies 2 files and basically updates unrelated comments as well as changes the platform registration, while leaving the deprecated method untouched.

ADeadPixel commented:
Maybe we were looking at different things but I removed the looping try catch system that used the deprecated async_forward_entry_setup method.

I replaced it with the async_forward_entry_setups that packages it all up and runs any platforms at once.

adamoutler commented:
Check the files tab in this PR.  There are no relevant changes.  https://github.com/adamoutler/anycubic-homeassistant/pull/14/files

ADeadPixel commented:
I'm not sure where your comment went but the most relevant changes are right here 
<img width="1408" alt="image" src="https://github.com/user-attachments/assets/14005544-5a8b-48bf-91e9-0222e6f7f5b3" /> on line 97.

adamoutler commented:
Try to make commits more narrow.  In this case you did make the required change and incremented version.  That's great!  But there were additional superfluous comment changes as well.  It's awesome that you continue to contribute to this project and I want to thank you for your support. You're doing great.

ADeadPixel commented:
I appreciate the kind words and feedback. I agree with you, looking back that the comments are not necessary and cloud up the PR. It is honestly a habit I've gotten into when I'm working on a file/method to "clean it up". Probably a hold over from my work where I'm reviving an abandoned code base and modernizing it. 

Either way, I really do appreciate your patience and feedback. Have a good one Adam :)
P.S. your personal site is down :( (mine is too haha)
